axelrod/tests/strategies/test_axelrod_second.py

```diff
@@ -490,3 +490,55 @@ def test_strategy(self):
                     [(D, C)] + [(C, C), (C, D), (D, C)] * 8 + \
                     [(D, C)] * 25
         self.versus_test(axelrod.WinShiftLoseStay(D), expected_actions=actions)
+
+class TestCave(TestPlayer):
+    name = "Cave"
+    player = axelrod.Cave
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        actions = [(C, C)] * 100 
+        self.versus_test(axelrod.Cooperator(), expected_actions=actions)
+
+        # It will take until turn 18 to respond decide to repond D->D
+        random.seed(1)
+        actions = [(C, D)]
+        actions += [(C, D), (D, D), (D, D), (C, D), (C, D), (C, D), (D, D),
+                    (D, D), (C, D), (C, D), (D, D), (C, D), (D, D), (C, D),
+                    (C, D), (D, D), (C, D)] # Randomly choose
+        actions += [(D, D)] * 30 # Defect
+        self.versus_test(axelrod.Defector(), expected_actions=actions)
```

Comment:
Can we put the seed argument in to `self.versus_test` please:

```
self.versus_test(axelrod.Defector(), expected_actions=actions, seed=1)
```

Same below.